package-lock.json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
We should add `package-lock.json` to the `.gitignore` file since we are preferring `yarn.lock` in our projects.